docs/md/modules-covid.md

```diff
@@ -0,0 +1,166 @@
+# Covid namespace
+
+WorkFlow Launcher (WFL) runs within the `covid` namespace to process workflows using the `Sarscov2IlluminaFull` pipeline. 
+
+Workflow Launcher uses a model which includes a source, a sink and an executor. Within the `covid` namespace, the source at the time of this writing is either directly from the Terra Data Repository (`TerraDataRepo`), or a list of snapshots which are passed to WFL (`TDR Snapshots`). 
+
+### API
+
+The `covid` module supports the following API endpoints:
+
+| Verb | Endpoint                            | Description                                                              |
+|------|-------------------------------------|--------------------------------------------------------------------------|
+| GET  | `/api/v1/workload`                  | List all workloads, optionally filtering by uuid or project              |
+| POST | `/api/v1/create`                    | Create a new workload                                                    |
+| POST | `/api/v1/start`                     | Start a workload                                                         |
+| POST | `/api/v1/stop`                      | Stop a running workload                                                  |
+| POST | `/api/v1/exec`                      | Create and start (execute) a workload                                    |
+
+Although it is possible to be run differently, the typical life-cycle of a workload is a multi-stage process:
```

Comment:
"Although it is possible to be run differently": is this caveat true for COVID?

Reply:
I don't think so. I think this is leftover from my attempt to generalize the module.